deploy/helm-chart/kubernetes-replicator/Chart.yaml

```
@@ -4,6 +4,6 @@ description: Controller for replicating secrets+configmaps across namespaces
 
 type: application
 
-version: 2.10.2
+version: 2.10.3
```

The chart and appVersion will be bumped automatically.

```suggestion
version: 2.10.2
```